server/server.go

```diff
@@ -4169,3 +4169,42 @@ func (s *Server) changeRateLimitLogInterval(d time.Duration) {
 	default:
 	}
 }
+
+// Disconnects a client by cid
+func (s *Server) DisconnectClientByID(id uint64) {
+	client := s.clients[id]
+	if client != nil {
+		client.closeConnection(Kicked)
+	}
+}
+
+// Disconnects a client by name
+func (s *Server) DisconnectClientsByName(name string) {
```

Comment:
maybe take this one out from the first pass, now I'm thinking that you could have same names under different accounts which could be an issue

Reply:
Took out potentially too powerful 'by name' functionality for both kick and ldm